src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx

```diff
@@ -0,0 +1,141 @@
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-octarine">
+                <img src={octarine} alt='Octarine Service Mesh' class="adapter-logo" />Meshery adapter for Octarine</a> 
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-nsm">
+                <img src={nsm} alt='Network Mesh' class="adapter-logo" />Meshery adapter for Network Service Mesh</a>
+            </td>
+        </tr>
+        <tr><td class="stable-adapters"></td></tr>
+        <tr>
+            <td rowspan="2" class="beta-adapters">beta</td>
+            <td><a href="https://github.com/layer5io/meshery-cpx">
+                <img src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQksHj15DkID308qQw3cmkQrRULPxyzbVquSZVev-9dj1L6sPs-rQ&s' alt='Citrix CPX Service Mesh' class="adapter-logo" />Meshery adapter for Citrix CPX</a>
+            </td>
+        </tr>
+        <tr><td class="beta-adapters"></td></tr>
+        <tr>
+            <td rowspan="6" class="alpha-adapters">alpha</td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-maesh">
+                <img src='https://raw.githubusercontent.com/containous/maesh/master/docs/content/assets/img/maesh.png' alt='Maesh Service Mesh' class="adapter-logo" />Meshery adapter for Maesh</a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-app-mesh">
+                <img src={awsappmesh} alt='AWS App Mesh Service Mesh' class="adapter-logo" />Meshery adapter for App Mesh </a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-kuma">
+                <img src='https://pbs.twimg.com/profile_images/1171109603158843392/bV2M7l39_400x400.jpg' alt='Kuma Service Mesh' class="adapter-logo" />Meshery adapter for Kuma</a>
+            </td>
+        </tr><tr>
+        </tr>
+        <tr><td class="alpha-adapters"></td></tr>
+        </tbody>
+    </table>
+</div>
+
+Some things that Linkerd is known for:
+
+<li>Purpose-built for Kubernetes</li>
+<li>Featuring custom-built, highly performant proxies written in Rust</li>
+<li>Zero-config option (works out-of-the-box)</li>
+<li>Network telemetry built-in (includes a pre-configured, optimised Prometheus instance)</li>
+<li>Low-overhead control-plane</li>
+<li>Operational simplicity (when compared to Istio, for example, even though Istio is getting better in this regard)</li>
+
+So what is covered in the video? More or less the following:
+
+<li>What service mesh tech allows us to do</li>
+<li>What a typical service mesh architecture looks like</li>
+<li>What Layer5 is about (<i>Lookout - it may surprise you!</i> 😯)</li>
+<li>What Meshery is. What Linkerd is.</li>
+<li>How easy it is to install Meshery on your PC (be it Linux, Mac or Windows)</li>
+<ul><li>All it takes is: 
+
+```sh
+$ curl -L https://git.io/meshery | bash - 
```

Comment:
The background is transparent for some reason.
![Screenshot from 2020-08-04 22-53-55](https://user-images.githubusercontent.com/42486853/89324753-6921eb80-d6a5-11ea-9863-20a26714182a.png)

Reply:
Strange. It works fine with me.

Reply:
This looks like a bigger problem. The background is not loading on refresh the page. Can you verify @Nikhil-Ladha

Reply:
Indeed it is @Tanuj22 initially the background loads, but then after a refresh ir page change it doesn't comes back. To confirm, I saw the blog from Kush as well `Getting started with mesheryctl` it also has a similar problem.

Reply:
This seems to be an issue with netlify, as the local site works perfectly on reload.

Reply:
As @Jashpatel1 said it is working locally. I think I have a solution for this in production. We can move ahead with this PR @Nikhil-Ladha . I will open a new PR for the solution

Reply:
Cool.